system_notification/infra/http/server/fastapi_http_server.py

```
@@ -0,0 +1,61 @@
+
+class FastApiHttpServer:
+    def __init__(self, server=uvicorn, app=FastAPI()) -> None:
+        self._app = app
+        self._server = server
+
+    def serve(self, port: int = 8000) -> None:
+        self._server.run(self._app, port=port)
+
+    def on(self, method: str, url: str, controller: Controller) -> None:
+        async def fastapi_controller(
+            request: Request, params: Dict[Any, Any] = Depends(get_query_params_as_dict)
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)